modelbaker/iliwrapper/ilicache.py

```diff
@@ -449,6 +458,12 @@ def process_model_directory(self, path):
         # download remote and local repositories
         self.download_repository(path)
 
+    def clear_cache():
+        if not QDir().exists(IliToppingFileCache.CACHE_PATH):
```

Comment:
It seems wrong to use `IliToppingFileCache` here. Shouln't it be `self` or at least `IliDataCache`?